docs/decisions/0004-decision-making.md

```diff
@@ -0,0 +1,25 @@
+# Decision making process
+
+* Status: accepted
+* Deciders: @rocketnova, @aplybeah, @microwavenby
+* Date: 2022-12-19
+
+Technical Story: https://wicmtdp.atlassian.net/browse/PRP-65
+
+## Context and Problem Statement
+
+How do we want to track significant technical decisions for this project?
+How do we want to share research and
+
+## Considered Options
+
+* Tech specs
+* ADRs
+* Tech specs + ADRs
+
+## Decision Outcome
+
+* We'll use ADRs in this github repo to track significant technical or architectural decisions. See 0000-use-markdown-architectural-decisions.md for why ADRs.
+* Any decisions that require significant research or discussion should first be documented as a Tech Spec in Confluence.
+
+Full process documented at: https://navasage.atlassian.net/wiki/spaces/MWDP/pages/553255058/Tech+specs+PRP
```

Comment:
I would love to see this process snapshotted in the GitHub repo (so that people reviewing this open-source project can follow how the decisions were / are being made)